### PR TITLE
TA-2894

### PR DESCRIPTION
**Pull request by JayVirdi**
## What does this pull request do?

Please describe what you did.
Added feature tests for YOUF, YOUK, and YOUL.

## Why make these changes?

Please describe why the changes were needed.
To ensure that we have adequate test coverage.

## Checklist

- [ ] Provided link to a JIRA ticket: https://dsdmoj.atlassian.net/browse/TA-2984

### CODE REVIEW

File: features/validations/crime_lower/youe_validations.feature [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Feature: YOUE code Bulk load validations
```
Comment (vimalsuresh) [inline features/validations/crime_lower/youe_validations.feature:1]: this test has both bulkload and manaul validation, can you amend the feature name please

File: features/validations/crime_lower/youf_validations.feature [hunk 2]
```diff
@@ -0,0 +1,25 @@
+Feature: YOUF code Bulk load validations
```
Comment (vimalsuresh) [inline features/validations/crime_lower/youf_validations.feature:1]: same as above

File: features/validations/crime_lower/youk_validations.feature [hunk 3]
```diff
@@ -0,0 +1,25 @@
+Feature: YOUK code Bulk load validations
```
Comment (vimalsuresh) [inline features/validations/crime_lower/youk_validations.feature:1]: same as above

File: features/validations/crime_lower/youl_validations.feature [hunk 4]
```diff
@@ -0,0 +1,25 @@
+Feature: YOUL code Bulk load validations
```
Comment (vimalsuresh) [inline features/validations/crime_lower/youl_validations.feature:1]: same as above

Comment (JayVirdi) [reply features/validations/crime_lower/youl_validations.feature]: I have pushed a change to fix these. Please re-review.

File: features/validations/crime_lower/youe_validations.feature (hunk 1 above)
Comment (solomonahmed) [reply features/validations/crime_lower/youe_validations.feature]: Ideally these are either in separate tests or tagged appropriately.

File: features/support/am/defaults/am_crime_lower_defaults.yml [hunk 5]
```diff
@@ -139,6 +139,28 @@ criminal_proceedings_base: &criminal_proceedings_base
   work_concluded_date:
   youth_court: N
 
+youth_court_base: &youth_court_base
```
Comment (solomonahmed) [inline features/support/am/defaults/am_crime_lower_defaults.yml:142]: Are the Youth Court codes within a separate category of law within Crime Lower?

File: features/support/am/fields/am_crime_lower.yml [hunk 6]
```diff
@@ -279,3 +299,17 @@ crime_lower:
       <<: *associated_civil_work_base
     ASAS:
       <<: *associated_civil_work_base
+  
+  youth_court:
+    YOUE:
```
Comment (solomonahmed) [inline features/support/am/fields/am_crime_lower.yml:304]: These codes will need to sit under criminal proceedings.  But, I will check with Janet on this.

**Review by solomonahmed — CHANGES_REQUESTED** [review]

File: features/validations/crime_lower/youe_validations.feature (hunk 1 above)
Comment (solomonahmed) [reply features/validations/crime_lower/youe_validations.feature]: Also, I wondered if we could start organising the tests into a folder structure which matches the categories of law within Crime Lower?  So youth court validation will sit in crime_lower/criminal_proceedings/youe_validations.feature.

Comment (JayVirdi) [reply features/validations/crime_lower/youe_validations.feature]: I have made the changes as requested, please re-review

**Review by solomonahmed — APPROVED** [review]
Great stuff @JayVirdi .

**Pull request merged**